plugins/node/opentelemetry-instrumentation-pg/src/utils.ts

```diff
@@ -236,3 +240,22 @@ export function patchCallbackPGPool(
     cb.call(this, err, res, done);
   };
 }
+
+export function patchConnectErrorCallback(
```

Comment:
Is this callback only for errors?

Reply:
Renamed it to `patchClientConnectCallback`